data/discovery/register/local-authority.yaml

```diff
@@ -0,0 +1,14 @@
+fields:
+- local-authority
+- parent-local-authority
```

Comment:
Please could you add uk to this list of fields to make consistent with the data in the local-authority-data repo.